tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -177,6 +177,15 @@ static int convertLLVMToSPIRV(const SPIRV::TranslatorOpts &Opts) {
   return 0;
 }
 
+static bool isFileEmpty(const std::string &FileName) {
+  std::ifstream file(FileName);
+  bool Result = false;
+  if (file && file.peek() == EOF)
+     Result = true;
+  file.close();
```

Comment:
Is it safe to call `close` if file wasn't opened at all? Will it be done automatically in `ifstream` destructor?

Reply:
I deleted this call, thanks.